### PR TITLE
refactor(nc-gui): reduce computed values generated by cells

### PR DESCRIPTION
**Pull request by bcakmakoglu**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [x] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nc-gui/components/smartsheet/Cell.vue [hunk 1]
```diff
@@ -139,36 +121,42 @@ const syncAndNavigate = (dir: NavigateDir) => {
 <template>
   <div
     class="nc-cell w-full h-full"
-    :class="{ 'text-blue-600': isPrimary && !virtual && !isForm }"
+    :class="{ 'text-blue-600': isPrimary(column) && !virtual && !isForm }"
```
Comment (pranavxc) [inline packages/nc-gui/components/smartsheet/Cell.vue:124]: `!virtual` can be removed and not at all available in the context

Comment (bcakmakoglu) [reply packages/nc-gui/components/smartsheet/Cell.vue:124]: It seems to be used by `Lookup.vue`

```vue
<LazySmartsheetCell :model-value="v" :column="lookupColumn" :edit-enabled="false" :virtual="true" />
```

Is the prop unnecessary then?

Comment (pranavxc) [reply packages/nc-gui/components/smartsheet/Cell.vue:124]: Ok it's required my bad

**Review by wingkwong — APPROVED** [review]
Resolve conflict then it's all good.

**Comment by o1lab** [comment]
@bcakmakoglu : pls resolve the conflicts and we can have it merged.

**Review by pranavxc — APPROVED** [review]

**Pull request merged**